gcc/rust/typecheck/rust-hir-type-check-base.cc

```diff
@@ -298,7 +298,7 @@ TypeCheckBase::parse_repr_options (const AST::AttrVec &attrs, location_t locus)
   repr.pack = 0;
   repr.align = 0;
 
-  // FIXME handle repr types....
+  // FIXME handle non-integer repr types....
```

Comment:
You can remove this FIXME and mention "enums default repr is  isize but we now check for other repr in the attibutes."